modules/storage-persistent-storage-fsGroup.adoc

```diff
@@ -31,5 +31,5 @@ securityContext:
 
 [NOTE]
 ====
-The fsGroupChangePolicyfield has no effect on ephemeral volume types, such as secret, configMap, and emptydir.
+The fsGroupChangePolicy field has no effect on ephemeral volume types, such as secret, configMap, and emptydir.
```

Comment:
```suggestion
The `fsGroupChangePolicy` field has no effect on ephemeral volume types, such as `secret`, `configMap`, and `emptyDir`.
```

Reply:
@dpateriya I think you missed to apply the suggested change. Can you please address it before the merge?